osu.ElasticIndexer/schemas/solo_scores.json

```diff
@@ -62,6 +62,9 @@
       },
       "user_id": {
         "type": "keyword"
+      },
+      "is_legacy": {
```

Comment:
should go between `id` and `max_combo`

Reply:
Done. Is this for style, or does it have a logical effect that I can test somehow?

Reply:
just style.